src/tox/tox_env/python/pip/pip_install.py

```diff
@@ -168,10 +168,7 @@ def build_install_cmd(self, args: Sequence[str]) -> List[str]:
             opts_at = install_command.index("{packages}")
         except ValueError:
             opts_at = len(install_command)
-        result = install_command[:opts_at]
-        result.extend(args)
-        result.extend(install_command[opts_at + 1 :])
-        return result
+        return install_command[:opts_at] + list(args) + install_command[opts_at + 1 :]
```

Comment:
again if you don't save the return value into a variable is a PITA to debug when something here is wrong

Reply:
This was not reverted, but set resolved? 🤔